modules/functions.js

```diff
@@ -0,0 +1,78 @@
+import { Book, main, booksList, bookListSection, sectionTitle } from './classes.js'; // eslint-disable-line
+
+export const populateAddBookSection = () => {
+  const addBookSection = document.createElement('section');
+  const addBookSectionTitle = document.createElement('h2');
+  const form = document.createElement('form');
+  addBookSection.id = 'add';
+  addBookSection.classList.add('add-book', 'd-flex', 'col', 'd-off');
+  addBookSectionTitle.textContent = 'Add a new book';
+  addBookSectionTitle.className = 'section-title';
+  form.id = 'create-form';
+  form.innerHTML = `<input name="title" type="text" placeholder="title" id="title" required>
+    <input name="author" type="text" placeholder="author" id="author" required>
+    <button type="submit" id="form-button" class="clickeable btn">Add</button>
+    <div id="success-msg"></div>`;
+
+  addBookSection.appendChild(addBookSectionTitle);
+  addBookSection.appendChild(form);
+
+  function saveFormData(book) {
```

Comment:
- [ ] As mentioned in the Project requirements we have to use ES6 syntax **"Refactor your methods and functions to arrow functions."** please update all your functions to use arrow functions syntax.

Reply:
done